pkg/schema/converter/k8stonorman.go

```diff
@@ -1,3 +1,5 @@
+// Package converter is responsible for converting the types registered with a k8s server to schemas which can be used
+// by the UI (and other consumers) to get a rough idea of the resources available and their current user's permissions
```

Comment:
```suggestion
// Package converter is responsible for converting the types registered with a k8s server to schemas which can be used
// by the UI (and other consumers) to discover the resources available and their current user's permissions.
```

Reply:
Added this fix and made another, related comment change.